src/SDL_SavePNG/savepng.h

```diff
@@ -7,16 +7,7 @@
  * http://www.libpng.org/pub/png/src/libpng-LICENSE.txt
  */
 #include <SDL_video.h>
-/*
- * Save an SDL_Surface as a PNG file.
- *
- * Returns 0 success or -1 on failure, the error message is then retrievable
- * via SDL_GetError().
- */
-#define SDL_SavePNG(surface, file) \
-	SDL_SavePNG_RW(surface, SDL_RWFromFile(file, "wb"), 1)
-//
-//TODO: filesystem::load_RWops is only for reading, would like a writing version also
+
```

Comment:
Do we really need this last commit?

This isn't our code, it's from https://github.com/driedfruit/SDL_SavePNG/blob/defbe6f77dace9cf4640e900f887cf8b022d55e3/savepng.h

I'd like to keep it as close to upstream as possible.

Reply:
This bit has been edited repeatedly in relation to our own RWops: https://github.com/wesnoth/wesnoth/commits/master/src/SDL_SavePNG/savepng.h